### PR TITLE
Modify project files to reduce warnings when building

### PR DESCRIPTION
**Pull request by nr-ahemsath**
### Description

Modifies project files in the unbounded int. tests solution to reduce build warnings.

### Testing

The solution builds cleanly with fewer warnings, and the tests still pass.

### Changelog

N/A

### CODE REVIEW

File: tests/Agent/IntegrationTests/UnboundedApplications/MongoDB2_6Application/MongoDB2_6Application.csproj [hunk 1]
```diff
@@ -104,11 +101,10 @@
       <Version>4.3.0</Version>
     </PackageReference>
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation">
-      <Version>4.0.0</Version>
+      <Version>4.3.0</Version>
```
Comment (jaffinito) [inline tests/Agent/IntegrationTests/UnboundedApplications/MongoDB2_6Application/MongoDB2_6Application.csproj:104]: Why the change in version here?

**Review by jaffinito — APPROVED** [review]
Other than my question about versions, this is good.

File: tests/Agent/IntegrationTests/UnboundedApplications/MongoDB2_6Application/MongoDB2_6Application.csproj (hunk 1 above)
Comment (nr-ahemsath) [reply tests/Agent/IntegrationTests/UnboundedApplications/MongoDB2_6Application/MongoDB2_6Application.csproj:104]: One class of build warnings was about package versions being downgraded.  Making this change eliminated the warning, and the tests still build and pass.
